### PR TITLE
Sage 300: Mapping Component rename and SubMenu - 1

### PR DESCRIPTION
**Pull request by anishfyle**

### CODE REVIEW

File: src/app/integrations/sage300/sage300-main/sage300-mapping/sage300-base-mapping/sage300-base-mapping.component.ts [hunk 1]
```diff
@@ -21,9 +19,9 @@ export class GenericMappingTestComponent implements OnInit {
 
   showAutoMapEmployee: boolean = false;
 
-  reimbursableExpenseObject: IntacctReimbursableExpensesObject | undefined;
+  // ReimbursableExpenseObject: IntacctReimbursableExpensesObject | undefined;
```
Comment (ashwin1111) [inline src/app/integrations/sage300/sage300-main/sage300-mapping/sage300-base-mapping/sage300-base-mapping.component.ts:22]: why are these commented?

Comment (ashwin1111) [reply src/app/integrations/sage300/sage300-main/sage300-mapping/sage300-base-mapping/sage300-base-mapping.component.ts:22]: whole file

File: src/app/integrations/sage300/sage300-main/sage300-mapping/sage300-mapping.component.ts [hunk 2]
```diff
@@ -7,9 +8,16 @@ import { Component, OnInit } from '@angular/core';
 })
 export class Sage300MappingComponent implements OnInit {
 
+  modules: MenuItem[] = [
+    {label: 'Employee Mapping', routerLink: '/integrations/sage300/main/mapping/employee'},
+    {label: 'Category Mapping', routerLink: '/integrations/intacct/main/configuration/category'},
+    {label: 'Generic Mapping', routerLink: '/integrations/intacct/main/configuration/generic'}
```
Comment (ashwin1111) [inline src/app/integrations/sage300/sage300-main/sage300-mapping/sage300-mapping.component.ts:14]: this should be removed

File: src/app/integrations/sage300/sage300-main/sage300-mapping/sage300-mapping.component.ts [hunk 3]
```diff
@@ -7,9 +8,16 @@ import { Component, OnInit } from '@angular/core';
 })
 export class Sage300MappingComponent implements OnInit {
 
+  modules: MenuItem[] = [
+    {label: 'Employee Mapping', routerLink: '/integrations/sage300/main/mapping/employee'},
+    {label: 'Category Mapping', routerLink: '/integrations/intacct/main/configuration/category'},
```
Comment (ashwin1111) [inline src/app/integrations/sage300/sage300-main/sage300-mapping/sage300-mapping.component.ts:13]: mapping/category

File: src/app/integrations/sage300/sage300-main/sage300-mapping/sage300-mapping.component.ts [hunk 4]
```diff
@@ -7,9 +8,16 @@ import { Component, OnInit } from '@angular/core';
...
+    {label: 'Generic Mapping', routerLink: '/integrations/intacct/main/configuration/generic'}
+  ];
```
Comment (ashwin1111) [inline src/app/integrations/sage300/sage300-main/sage300-mapping/sage300-mapping.component.ts:15]: there should be API call to get all mapping settings - particularly for generic mappings

**Review by ashwin1111 — APPROVED** [review]

File: src/app/integrations/sage300/sage300-main/sage300-mapping/sage300-mapping.component.ts (hunk 4 above)
Comment (anishfyle) [reply src/app/integrations/sage300/sage300-main/sage300-mapping/sage300-mapping.component.ts:15]: covered in #265

**Pull request merged**